data/campaigns/Heir_To_The_Throne/scenarios/17_Scepter_of_Fire.cfg

```diff
@@ -6,7 +6,7 @@
     map_generation=lua
     [generator]
         id="cavegen"
-        config_name="Lua Cave Generator"
+        config_name=_"Sceptre of Fire Caves"
```

Comment:
Where is this shown?

Reply:
I don't think it's shown anywhere, because this file is not parsed in map editor mode.

If it _was_ parsed in map editor mode, you would see it in the Generate Map dialog.